services/core-web/src/components/mine/Reports/MineReportTable.js

```diff
@@ -173,6 +173,7 @@ export const MineReportTable = (props) => {
                 openEditReportModal={record.openEditReportModal}
                 handleEditReport={record.handleEditReport}
                 handleRemoveReport={record.handleRemoveReport}
+                minePermitRecord={record}
               />
```

Comment:
Can you rename this prop to be related to reports not permits? (`reportRecord={record}`)

Reply:
Thanks a lot  for this requesting this change and others discussed in teams. This prop is removed. After the changes is no longer used.